README.rst

```diff
@@ -9,7 +9,8 @@ of `Kafka REST <https://docs.confluent.io/platform/current/kafka-rest/index.html
 
 |Tests| |Contributor Covenant|
 
-.. |Tests| image:: https://github.com/aiven/karapace/actions/workflows/tests.yml/badge.svg
+.. |Tests| image:: https://github.com/aiven/karapace/actions/workflows/tests.yml/badge.svg?branch=main
+    :target: https://github.com/aiven/karapace/actions/workflows/tests.yml
```

Comment:
suggestion: I think we should preferably select main in the link as well, ie use `https://github.com/aiven/karapace/actions/workflows/tests.yml?query=branch%3Amain`.